modules/twinklefluff.js

```diff
@@ -75,126 +75,136 @@ Twinkle.fluff = {
 
-				var agfLink = document.createElement('a');
-				var vandLink = document.createElement('a');
-				var normLink = document.createElement('a');
+			agfLink.href = "#";
+			vandLink.href = "#";
+			normLink.href = "#";
+			$(agfLink).click(function(){
+				Twinkle.fluff.revert('agf', vandal);
+			});
+			$(vandLink).click(function(){
+				Twinkle.fluff.revert('vand', vandal);
+			});
+			$(normLink).click(function(){
+				Twinkle.fluff.revert('norm', vandal);
+			});
 
-				agfLink.href = "#";
-				vandLink.href = "#";
-				normLink.href = "#";
-				$(agfLink).click(function(){
-					Twinkle.fluff.revert('agf', vandal);
-				});
-				$(vandLink).click(function(){
-					Twinkle.fluff.revert('vand', vandal);
-				});
-				$(normLink).click(function(){
-					Twinkle.fluff.revert('norm', vandal);
-				});
+			agfLink.appendChild( spanTag( 'Black', '[' ) );
+			agfLink.appendChild( spanTag( 'DarkOliveGreen', 'rollback (AGF)' ) );
+			agfLink.appendChild( spanTag( 'Black', ']' ) );
 
-				agfLink.appendChild( spanTag( 'Black', '[' ) );
-				agfLink.appendChild( spanTag( 'DarkOliveGreen', 'rollback (AGF)' ) );
-				agfLink.appendChild( spanTag( 'Black', ']' ) );
+			vandLink.appendChild( spanTag( 'Black', '[' ) );
+			vandLink.appendChild( spanTag( 'Red', 'rollback (VANDAL)' ) );
+			vandLink.appendChild( spanTag( 'Black', ']' ) );
 
-				vandLink.appendChild( spanTag( 'Black', '[' ) );
-				vandLink.appendChild( spanTag( 'Red', 'rollback (VANDAL)' ) );
-				vandLink.appendChild( spanTag( 'Black', ']' ) );
+			normLink.appendChild( spanTag( 'Black', '[' ) );
+			normLink.appendChild( spanTag( 'SteelBlue', 'rollback' ) );
+			normLink.appendChild( spanTag( 'Black', ']' ) );
 
-				normLink.appendChild( spanTag( 'Black', '[' ) );
-				normLink.appendChild( spanTag( 'SteelBlue', 'rollback' ) );
-				normLink.appendChild( spanTag( 'Black', ']' ) );
+			agfNode.appendChild(agfLink);
+			vandNode.appendChild(vandLink);
+			normNode.appendChild(normLink);
 
-				agfNode.appendChild(agfLink);
-				vandNode.appendChild(vandLink);
-				normNode.appendChild(normLink);
+			revertNode.appendChild( agfNode );
+			revertNode.appendChild( document.createTextNode(' || ') );
+			revertNode.appendChild( normNode );
+			revertNode.appendChild( document.createTextNode(' || ') );
+			revertNode.appendChild( vandNode );
 
-				revertNode.appendChild( agfNode );
-				revertNode.appendChild( document.createTextNode(' || ') );
-				revertNode.appendChild( normNode );
-				revertNode.appendChild( document.createTextNode(' || ') );
-				revertNode.appendChild( vandNode );
+			ntitle.insertBefore( revertNode, ntitle.firstChild );
+		}
+	},
 
-				ntitle.insertBefore( revertNode, ntitle.firstChild );
-			}
+	normal: function() {
+		if( mw.config.get('wgNamespaceNumber') === -1 && mw.config.get('wgCanonicalSpecialPageName') === "Contributions" ) {
+			Twinkle.fluff.contributions();
+		} else {
+			mw.hook( 'wikipage.diff' ).add( function () {
+				this.diff();
```

Comment:
```suggestion
				Twinkle.fluff.diff();
```
@WMDE-Fisch 	This too